pkg/cmd/cli/secrets/new.go

```diff
@@ -71,7 +72,7 @@ func NewCmdCreateSecret(name, fullName string, f *clientcmd.Factory, out io.Writ
 	}
 
 	cmd.Flags().BoolVarP(&options.Quiet, "quiet", "q", options.Quiet, "Suppress warnings")
-	cmd.Flags().VarP(&options.Sources, "source", "f", "List of filenames or directories to populate the data elements in a secret")
+	cmd.Flags().VarP(&options.Sources, "source", "f", "List of filenames or directories to populate keys in the secret.")
```

Comment:
@deads2k [this](https://github.com/openshift/origin/pull/3011/files#diff-9813e072958dea3fd13e14e8c69e9255R55) look OK?